tests/sentry/grouping/grouping_inputs/cocoa_dispatch_client_callout.json

```diff
@@ -0,0 +1,114 @@
+{
+  "platform": "cocoa",
+  "message": "Foo",
+  "threads": {
+    "values": [
+      {
+        "stacktrace": {
+          "frames": [
```

Comment:
I've added enough frames to make sure the customer request gets fixed.

[This commit](https://github.com/getsentry/sentry/pull/61675/commits/deca389dd41bda50d6c15b83fcfe031e2c14173b) shows what happens when the rule is in place.

`system` and `app` do not get any hashes, thus, grouping by the message.
<img width="262" alt="image" src="https://github.com/getsentry/sentry/assets/44410/263b23cf-837d-44da-945b-0a5eb8f7b7d7">